RELEASE_NOTES.md

```diff
@@ -2,6 +2,7 @@
 
 - fix edge condition when calling `Connection#commit()` threw an Exception, we called commit() again. Now the
   code explicitly calls `Connection#rollback()`. Fixes #2595
+- FieldMapper: skip static fields (#2607)
```

Comment:
I started to explicitly call out the people that reported bugs. This gives them a mention on the release page (e.g. here: https://github.com/jdbi/jdbi/releases/tag/v3.43.0) so they can take that back to their managers to justify contributing to Jdbi in their work time. It's a tiny thing for us but I know that some organizations track metrics for that.

Reply:
I am happy to give collaborators attribution where due, but doing it for the corporate metrics seems like the wrong reason to do it to me... we would encourage tiny pointless contributions to boost your metrics

Reply:
![image](https://github.com/jdbi/jdbi/assets/129097/1cca6c49-300c-4ae0-8743-d98fc15251ee)